### ISSUE TITLE
Apparent magnitude of a star for a given moment?

### ISSUE THREAD

**Issue by gloglo17** [skyfielders/python-skyfield#853]
Wikipedia: "Algol's apparent magnitude is usually near-constant at 2.1, but regularly dips to 3.4 every 2.86 days during the roughly 10-hour-long partial eclipses."

Is there a way how to get the apparent magnitude of a specific star for a given moment and location? It would be really helpful.

**Comment by brandon-rhodes** [comment]
No, I don't think there's anything in Skyfield that would understand a star with variable magnitude or be able to accurately create its light curve. A quick Google search for "python variable star magnitude" brings up packages that let you do light-curve analysis if you have been regularly observing a star, but the first page of results, at least, doesn't seem to include anything that goes in the other direction, and predicts future magnitudes.

I'll mark this issue as a "feature request" in case anyone can find prior work that could be adapted to become Skyfield code. For example, does something like Stellarium support Algol and make it different brightnesses on different nights? A quick search I just did wasn't encouraging, but someone else might be able to find working code for variable stars.

**Comment by gloglo17** [comment]
In ancient Egypt, they had a calendar of lucky and unlucky days, which alternated roughly every 2.85 days. According to researchers, this may have been derived from the orbital plane of Algol (2.867 days). It would be very interesting to have a function to calculate those variable magnitudes to check if there is something to it :))
